sql_generators/stable_views/__init__.py

```diff
@@ -197,19 +197,26 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, schema: SchemaF
         with metadata_file.open("w") as f:
             f.write(metadata_content)
 
-    # get view schema with descriptions
-    try:
-        content = VIEW_CREATE_REGEX.sub("", target_file.read_text())
-        content += " WHERE DATE(submission_timestamp) = '2020-01-01'"
-        view_schema = Schema.from_query_file(target_file, content=content)
+    if schema.user_facing_view not in {
+        # skip main and its clones because their large schema causes frequent API
+        # failures when trying to update the view schema
+        "telemetry.main",
+        "telemetry.first_shutdown",
+        "telemetry.saved_session",
```

Comment:
Should we introduce some kind of `SKIP` list (like we have in other places)?

Reply:
sure, I'll move it to `SKIP_VIEW_SCHEMA`